language-support/hs/bindings/examples/nim-console/src/UI.hs

```diff
@@ -70,7 +70,7 @@ parseWords = \case
     ["show"] ->
         return $ Query ShowOpenState
     ["offer"] ->
-        return $ Submit Local.OfferNewGameToAnyone
+        Nothing -- return $ Submit Local.OfferNewGameToAnyone
```

Comment:
Deliberate? Or commented out by mistake?

Reply:
Deliberate. 
I didn't want to accidentally forget to type the parties when I demoed.

The Anyone is currently a hardcoded hack.
Will reinstate it when a central registration mechanism for players is implemented in the DAML.